src/Aspire.Hosting/ApplicationModel/CustomResourceSnapshot.cs

```diff
@@ -27,6 +27,16 @@ public sealed record CustomResourceSnapshot
     /// </summary>
     public DateTime? CreationTimeStamp { get; init; }
```

Comment:
We already have a creation time stamp which is different from the startup timestamp. The creation timestamp is displayed in the "Start time" column. Seems odd. Wouldn't the `StartupTimestamp` make more sense?

Reply:
I think that makes more sense, yes